### ISSUE TITLE
Allow payment method terms to be hidden or customizable

### ISSUE THREAD

**Issue by kaushikasomaiya** [woocommerce/woocommerce-gateway-stripe#3680]
9178567-zen

> By providing your card information, you allow "name of business" to charge your card for future payments in accordance with their terms.

Merchants might want to display the terms their own way. I am not too sure but it seems it might be possible to control the visibility of the terms by adjusting the [terms object](https://docs.stripe.com/js/elements_object/create_payment_element#payment_element_create-options-terms) values if it is not possible to allow text customization. This way merchants can customize and display their terms by other ways. (for instance hooking custom text below the checkout button)

**Comment by ara303** [comment]
Just adding my :+1: to this. 

It's nitpicky but terms text is awkwardly worded. It says "in accordance with their terms," but so far as the customer's concerned shouldn't it be "our" terms? It refers to the store by name but then as if it's an external entity. I don't think Stripe provides a way to modify the wording but removing the text is possible.

In theory this should be as simple as attaching `card = 'never'` to [`terms`](https://docs.stripe.com/js/elements_object/create_payment_element#payment_element_create-options-terms), but I looked exhaustively and couldn't find any convenient way to amend that object. 

I also tried to work backwards using what is exposed as part of [the Elements Appearance API](https://woocommerce.com/document/stripe/customization/style-payment-form/#elements-appearance-api) but that seems to be strictly for working with style, couldn't change this through that.

Could a filter be added so we can amend the `options`, please?

**Comment by kaushikasomaiya** [comment]
So I think one solution could be adding a custom parameter to force disable the terms in 

https://github.com/woocommerce/woocommerce-gateway-stripe/blob/bcb76737a6b219156b8f77afb9317864d07c1df9/includes/payment-methods/class-wc-stripe-upe-payment-gateway.php#L370 

which can be modified through the filter `wc_stripe_upe_params`

and add some logic here to accommodate that

https://github.com/woocommerce/woocommerce-gateway-stripe/blob/bcb76737a6b219156b8f77afb9317864d07c1df9/client/stripe-utils/utils.js#L266

**Comment by ara303** [comment]
The ability to customise the text underneath the card info inputs would be really cool in my opinion. I'd definitely still like either the ability to a.) modify the text, or b.) hide it.

Edit for clarity: Some Stripe docs seem to suggest this text needs to be there for recurring payments (like WooCommerce Subscriptions), so I'd like to clarify I'd prefer to be able to modify it over hiding.

**Comment by ericboles** [comment]
Agree--the wording on this text is quite confusing for a customer since it sounds like it's coming from a third-party--especially with the "their terms" wording.  We should have the ability to customize this or remove it.

**Comment by ara303** [comment]
I'd still like customisation over this!
